MediaBrowser.Controller/Library/IUserDataManager.cs

```diff
@@ -26,13 +26,11 @@ public interface IUserDataManager
         /// <param name="userData">The user data.</param>
         /// <param name="reason">The reason.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
-        /// <returns>Task.</returns>
         void SaveUserData(Guid userId, BaseItem item, UserItemData userData, UserDataSaveReason reason, CancellationToken cancellationToken);
         void SaveUserData(User userId, BaseItem item, UserItemData userData, UserDataSaveReason reason, CancellationToken cancellationToken);
 
         UserItemData GetUserData(User user, BaseItem item);
 
-        UserItemData GetUserData(string userId, BaseItem item);
```

Comment:
why did you remove this helper btw?

Reply:
It encourages bad practice.

Reply:
I don't understand why it is so. We're receiving this parameter as a string from the outside world, what's so bad in passing it further in a helper function?

Reply:
It’s a 1 function call to convert it to a guid, and we are only receiving it as a string cause the current api code doesn’t convert it to a guid by itself, ASP.net core can.